deployment/scripts/1-ubuntu-dependencies.sh

```diff
@@ -11,35 +14,55 @@ if [ -z $(which docker) ]; then
     fi
     sudo apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
     sudo usermod -aG docker $(whoami)
+    if [ $? -eq 0 ]; then
```

Comment:
this would be just testing the "usermod" command for success/fail
testing the output of the apt-get install is more interesting

Reply:
ok changing